### PR TITLE
[PDI-13557] As a Hadoop/PDI developer I want to know about internal H…

### PR DESCRIPTION
**Pull request by SergeyTravin**
…adoop node information in a mapper, reducer or combiner transformation.

Move dublicated code to MRUtil.Class

From: Miguel Gaspar [mailto:miguel.gaspar@webdetails.pt]
Sent: Friday, April 24, 2015 7:56 AM
To: Aliaksandr Drebenchuk
Cc: Kanstantsin Karneliuk
Subject: Re: PDI-13557

Hi I am traveling, and it's hard to get internet.

Similar code, line 151:
https://github.com/pentaho/pentaho-hadoop-shims/blob/master/common/src-mapred/org/pentaho/hadoop/mapreduce/PentahoMapRunnable.java

### CODE REVIEW

**Comment by SergeyTravin** [comment]
@mfgaspar, @mattyb149, @e-cuellar
Could you review this PR. Please see https://github.com/pentaho/pentaho-hadoop-shims/pull/219

**Comment by mfgaspar** [comment]
Fine by me!

**Comment by kolinus** [comment]
@mattyb149 @brosander could you please merge?

File: common/src-mapred/org/pentaho/hadoop/mapreduce/MRUtil.java [hunk 1]
```diff
@@ -108,6 +110,35 @@ private static void initKettleEnvironment(Configuration conf) throws KettleExcep
     }
   }
 
+  public static void passInformationToTransformation(final VariableSpace variableSpace, final JobConf job) {
+    variableSpace.setVariable("Internal.Hadoop.NumMapTasks", Integer.toString(job.getNumMapTasks()));
```
Comment (mattyb149) [inline common/src-mapred/org/pentaho/hadoop/mapreduce/MRUtil.java]: Needs a check for variableSpace != null

**Comment by SergeyTravin** [comment]
@mattyb149 @brosander could you please merge?

**Comment by kcruzada** [comment]
@mattyb149 @brosander @mdamour1976 can you process this pull request?

**Pull request merged**